corehq/apps/adm/README.md

```diff
@@ -1,17 +0,0 @@
-# Active Data Management Reports
-
-## Enabling Projects
-
-You can enable projects to view ADM reports in the Reports section of HQ
-by adding the following document to your couchdb:
-
-```{
-    "_id": "ADM_ENABLED_DOMAINS",
-    "domains": []
-}```
```

Comment:
I just checked this doc on prod and `crs-remind` is also listed. Can we check their usage?